examples/tokenvm/DEVNETS.md

```diff
@@ -75,14 +75,15 @@ deployment target. If you use Graivtron CPUs, make sure to use `arm64` here.
 Now we can spin up a new network of 6 nodes with some defaults:
 - `avalanche-ops` supports [Graviton-based processors](https://aws.amazon.com/ec2/graviton/) (ARM64). Use `--arch-type arm64` to run nodes in ARM64 CPUs.
 - `avalanche-ops` supports [EC2 Spot instances](https://aws.amazon.com/ec2/spot/) for cost savings. Use `--instance-mode=spot` to run instances in spot mode.
+- `avalanche-ops` supports multi-region deployments. Use `--auto-regions 3` to distribute node across 3 regions.
```

Comment:
Can you add the example flag for listing specific regions too?

Reply:
Added a note